.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+name: Handle stale issues and PRs
+on:
+  schedule:
+    - cron: "0 11 * * *"
+  push:
+    branches: [stale-bot]
```

Comment:
```suggestion
```

Reply:
I am guessing https://github.com/astronomer/astronomer-cosmos/issues/1129#issuecomment-2446333216 was result of the push to this branch?